### PR TITLE
7413 appliance governance

### LINKED ISSUE #7413: [Core appliances pages] Build Governance
Build the governance at /appliance/governance from the design and with the [copy doc](https://drive.google.com/open?id=1DpU0AzAJ4gBa_au4gELBUD3vLz8l-FnAwFdERBHlvwc).

### PR DESCRIPTION
**Pull request by deadlight**
Pair coded with @anthonydillon 

## Done

- Updated the /appliance/governance page content
- Added an intitial contact us page (will need to update the details when the real one is created)
- Added the second-level nav for the appliance section

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/governance
- http://0.0.0.0:8001/appliance/contact-us
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes #7413
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2715

### CODE REVIEW

**Comment by deadlight** [comment]
@anthonydillon push your changes up to this PR

File: templates/appliance/governance.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      </ul>
+      <p>In the docs section, you will find access to the image builder. A tool where anyone can build appliance images out of whichever snaps they please to test on their own. Any images this way are not guaranteed to work and are also considered experimental.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      Hardware compatibility
```
Comment (pmahnke) [inline templates/appliance/governance.html:105]: This should be an h2?

File: templates/appliance/governance.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      Ubuntu Core is guaranteed to work on any Ubuntu certified hardware. This includes Raspberry Pi’s and the Intel NUC. Certification means that these boards exist in Canonical’s certification and testing lab and every time an update, bug fix or security patch is rolled out to the OS it is tested on these boards first. This is to ensure no update breaks a user’s appliance. Before a project can become an official Ubuntu appliance, it must be tested to work on a Raspberry Pi or Intel NUC.
```
Comment (pmahnke) [inline templates/appliance/governance.html:106]: Ubuntu certified hardware should be a link

File: templates/appliance/governance.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Decision-making bodies</h2>
+      <p>In order to have your project published as an official Ubuntu appliance and featured in the portfolio, you must get in touch with the right people at Canonical. The easiest way to do this is through the snapcraft forum, where Canonical engineers can help you package your software as a snap and test your images. It is then a simple case of creating a post in the snapcraft forum or making a topic in the Ubuntu Discourse. After that, the decision rests with the Ubuntu Community Council. The primary community (i.e., non-technical) governance body for the Ubuntu project. Members are appointed by Mark Shuttleworth and approved by a vote by the entire Ubuntu membership.</p>
```
Comment (pmahnke) [inline templates/appliance/governance.html:115]: missing 3 links

File: templates/appliance/governance.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+      <h3>Ubuntu Community Council</h3>
+      <p>The social structures and community processes of Ubuntu are supervised by the Ubuntu Community Council, which manages nominations and elections for Ubuntu boards and councils. The council is also responsible for the Code of Conduct and tasked with ensuring that community members and leaders live up to the standard it sets.</p>
```
Comment (pmahnke) [inline templates/appliance/governance.html:117]: missing two links

File: templates/appliance/governance.html [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+      <p>The Council is ultimately responsible for dispute resolution, should it be required. For example, in the past, we have helped to resolve conflicts in LoCo teams and in the Ubuntu forums - both very important parts of the community that have their own leadership structures carrying authority delegated by the Community Council.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h2>Commercial terms</h2>
+      <p>In order to use an Ubuntu appliance image in a commercial product, you must <a href="">contact Canonical</a> so that we can talk to the developer about the use of their work and Ubuntu Core in your product. You must see the appliances and Canonical’s licensing terms before using the image in any commercial product.</p>
```
Comment (pmahnke) [inline templates/appliance/governance.html:127]: link's to no where? I think the comments says there is a page?  For now you could link to /core/contact-us?

File: templates/appliance/governance.html [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+      </div>
+    </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h2>Code of Conduct</h2>
+      <p>One of the key innovations that Ubuntu pioneered in free software communities is the use of a Code of Conduct. It lays out the expectations that we have of those who participate, represent or engage with the project. Essentially, it calls on people to treat one another with respect regardless of their differences of opinion, and it is the foundation for all of our governance practices. The Ubuntu Code of Conduct continues to serve as a template for many other communities since it was established in 2004.</p>
```
Comment (pmahnke) [inline templates/appliance/governance.html:39]: missing link on Code of Conduct

File: templates/appliance/governance.html [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Privacy Policy</h2>
+      <p>Each appliance in the portfolio is expected to operate under European GDPR or California CCPA laws. Further than this, each appliance must provide information regarding their specific privacy policies on their dedicated appliance page. This should be completely transparent about what information is collected and who receives it.</p>
```
Comment (pmahnke) [inline templates/appliance/governance.html:48]: Missing 2 links

File: templates/appliance/governance.html [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+      <p>An appliance that is found to intentionally disregard these requirements will be removed from the portfolio and their breach of governance will be communicated to their users if possible, or otherwise made public knowledge.</p>
+      <p>The privacy policy associated with Canonical remains the same.</p>
```
Comment (pmahnke) [inline templates/appliance/governance.html:50]: Missing link

**Comment by anthonydillon** [comment]
@pmahnke Sorry this is missing my changes for some reason

**Comment by anthonydillon** [comment]
@pmahnke pushed my changes to the branch again and it worked.

File: templates/appliance/governance.html [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Decision-making bodies</h2>
+      <p>In order to have your project published as an official Ubuntu appliance and featured in the portfolio, you must get in touch with the right people at Canonical. The easiest way to do this is through the <a href="https://forum.snapcraft.io/categories" class="p-link--external">snapcraft forum</a>, where Canonical engineers can help you package your software as a snap and test your images. It is then a simple case of creating a post in the snapcraft forum or making a topic in the <a href="https://discourse.ubuntu.com/">Ubuntu Discourse</a>. After that, the decision rests with the <a href="https://docs.google.com/document/d/1DpU0AzAJ4gBa_au4gELBUD3vLz8l-FnAwFdERBHlvwc/edit#heading=h.ifg3ehffcmkc" class="p-link--external">Ubuntu Community Council</a>. The primary community (i.e., non-technical) governance body for the Ubuntu project. Members are appointed by Mark Shuttleworth and approved by a vote by the entire Ubuntu membership.</p>
```
Comment (pmahnke) [inline templates/appliance/governance.html:115]: you are linking to the copy doc here

File: templates/appliance/governance.html [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Privacy Policy</h2>
+      <p>Each appliance in the portfolio is expected to operate under <a href="https://gdpr-info.eu/" class="p-link--external">European GDPR</a> or <a href="https://oag.ca.gov/privacy/ccpa" class="p-link--external">California CCPA laws</a>. Additionally, each appliance must provide information regarding their specific privacy policies on their dedicated appliance page. It should be completely transparent about what information is collected and who receives it.</p>
+      <p>If an appliance is found to have intentionally disregarded these requirements will be removed from the portfolio and their breach of governance will be communicated to their users if possible, or otherwise made public knowledge.</p>
+      <p>The <a href="https://ubuntu.com/legal/data-privacy">privacy policy associated with Canonical</a> remains the same.</p>
```
Comment (pmahnke) [inline templates/appliance/governance.html:50]: remove the https://ubuntu.com bit

File: templates/appliance/governance.html (hunk 9 above)
Comment (pmahnke) [inline templates/appliance/governance.html:115]: is discourse external? also linking to the copy doc

File: templates/appliance/governance.html (hunk 10 above)
Comment (pmahnke) [inline templates/appliance/governance.html:50]: remove the 'https://ubuntu.com

File: templates/appliance/governance.html (hunk 9 above)
Comment (deadlight) [reply templates/appliance/governance.html:115]: It's on a completely different URL in this instance

Comment (deadlight) [reply templates/appliance/governance.html:115]: Oops... also it's an in-page link to the next line of text. I've just removed it.

File: templates/appliance/governance.html (hunk 5 above)
Comment (anthonydillon) [reply templates/appliance/governance.html]: It does now

File: templates/appliance/governance.html (hunk 6 above)
Comment (anthonydillon) [reply templates/appliance/governance.html]: Added link

File: templates/appliance/governance.html (hunk 9 above)
Comment (anthonydillon) [reply templates/appliance/governance.html]: We have started to remove external links to the ubuntu discourse as it is integrated via the header. Even though technically this is another domain for the user it shouldn't. And, there is a native navigation path back to the main site.

**Pull request merged**